tsconfig.json

```diff
@@ -0,0 +1,112 @@
+{
```

Comment:
Is this also needed?

Reply:
Yes I think it should be there. Its the Typescript config file

Reply:
I got the explaination from https://www.digitalocean.com/community/tutorials/typescript-new-project# 
Using the --init flag in the above command will initialize your project by creating a tsconfig.json file in your typescript-project project directory. This tsconfig.json file will allow you to configure further and customize how TypeScript and the tsc compiler interact. You can remove, add, and change configurations in this file to best meet your needs.

Reply:
Ok, but we are not using the typescript compiler for now, right? Maybe we should add it later.

Reply:
okay, deleted it